
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


cat: write error: Broken pipe
2023-01-26 21:12:24.447433
2023-01-26 21:12:25.417839
2023-01-26 21:12:26.330768

real	0m2.869s
user	0m2.654s
sys	0m0.181s
cat: write error: Broken pipe
2023-01-26 21:12:37.546030
2023-01-26 21:12:38.431048
2023-01-26 21:12:39.234630

real	0m2.622s
user	0m2.447s
sys	0m0.151s
cat: write error: Broken pipe
2023-01-26 21:12:50.432069
2023-01-26 21:12:51.178863
2023-01-26 21:12:51.919026

real	0m2.394s
user	0m2.229s
sys	0m0.138s
cat: write error: Broken pipe
2023-01-26 21:13:03.168906
2023-01-26 21:13:03.965776
2023-01-26 21:13:04.692155

real	0m2.484s
user	0m2.288s
sys	0m0.167s
cat: write error: Broken pipe
2023-01-26 21:13:16.017939
2023-01-26 21:13:16.917878
2023-01-26 21:13:17.788296

real	0m2.813s
user	0m2.605s
sys	0m0.177s
cat: write error: Broken pipe
2023-01-26 21:13:29.158176
2023-01-26 21:13:32.302447
2023-01-26 21:13:35.658651

real	0m7.578s
user	0m7.081s
sys	0m0.461s
cat: write error: Broken pipe
2023-01-26 21:13:46.906447
2023-01-26 21:13:48.188550
2023-01-26 21:13:50.529912

real	0m4.565s
user	0m4.244s
sys	0m0.286s
cat: write error: Broken pipe
2023-01-26 21:14:01.760046
2023-01-26 21:14:02.157462
2023-01-26 21:14:02.432990

real	0m1.618s
user	0m1.455s
sys	0m0.139s
mkdir: cannot create directory ‘SCODE_D_2’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_4’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_6’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_8’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
/data/gpfs/projects/punim0638/stephenz/miniconda3/lib/python3.10/site-packages/Scribe-0.0.0-py3.10.egg/Scribe/causal_network.py:18: UserWarning:  WARNING: No expression data argument given. if you intend to load the data from a file, call the method 'read_expression_file'.
  if expression==None: warnings.warn(" WARNING: No expression data argument given. if you intend to load the data from a file, call the method 'read_expression_file'.")
/data/gpfs/projects/punim0638/stephenz/miniconda3/lib/python3.10/site-packages/Scribe-0.0.0-py3.10.egg/Scribe/causal_network.py:298: FutureWarning: Could not cast to int64, falling back to object. This behavior is deprecated. In a future version, when a dtype is passed to 'DataFrame', either all columns will be cast to that dtype, or a TypeError will be raised.
  max_rdi_delay = pd.DataFrame({node_id: [np.nan for i in self.node_ids] for node_id in self.node_ids}, index=self.node_ids,
Create causal_model successfully
  Activating project at `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-BFC/dyn-BFC-1000-8/pidc_env`
    Updating registry at `~/.julia/registries/General.toml`
   Resolving package versions...
  No Changes to `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-BFC/dyn-BFC-1000-8/pidc_env/Project.toml`
  No Changes to `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-BFC/dyn-BFC-1000-8/pidc_env/Manifest.toml`
Getting nodes...
Inferring network...
